packages/breez_sdk_liquid/lib/src/breez_sdk_liquid.dart

```diff
@@ -120,10 +120,11 @@ class BreezSDKLiquid {
           _paymentEventStream.add(PaymentEvent.fromSdkEvent(event));
         } else if (event is liquid_sdk.SdkEvent_PaymentFailed) {
           _paymentEventStream.addError(event);
-        } else if (event is liquid_sdk.SdkEvent_Synced) {
-          _didCompleteInitialSyncController.add(null);
         }
         await _fetchWalletData(sdk);
+        if (event is liquid_sdk.SdkEvent_Synced) {
+          _didCompleteInitialSyncController.add(null);
+        }
```

Comment:
So the fix here is fetching the wallet info/payments before marking the initial sync complete?